cdap-security/src/main/java/co/cask/cdap/security/store/FileSecureStore.java

```diff
@@ -102,12 +100,13 @@
    * @param name Name of the element to store.
    * @param data The data that needs to be securely stored.
    * @param description User provided description of the entry.
-   * @param properties Metadata associated with the data  @throws IOException
+   * @param properties Metadata associated with the data
+   * @throws IOException If it failed to store the key in the store.
    */
   @Override
   public void put(String namespace, String name, byte[] data, String description, Map<String, String> properties)
```

Comment:
should handle the exception cases (not found/already exists, etc) in this class too.